packages/core/index.d.ts

```diff
@@ -61,6 +61,7 @@ declare module '@rjsf/core' {
         onChange: (formData: T, newErrorSchema: ErrorSchema) => void;
         onBlur: (id: string, value: boolean | number | string | null) => void;
         submit: () => void;
+        formElement: HTMLFormElement | null;
```

Comment:
Why add this? I thought this was meant to be private -- we don't have it documented -- but perhaps I'm wrong, is there a reason why someone would need to access this property?

Reply:
ahh, if it should be private, i'll hide that.  I had been using it undocumented in a project,  but can easily remove that

Reply:
Oh -- how were you using it? If people are already using it then we might as well just document it

Reply:
I'm probably doing something a bit non-ideal with it. 

in an onError handler i'm looking up the property path of the field with the first error, then converting that to the name string, and finding the formElement child that matches to focus on that field.  it's non-ideal, but I'm not aware of another way to do something similar yet

Reply:
I was using it previously to attach event handlers directly to the form for specific purposes and to avoid composing callbacks.
Are you sure that `formElement` can be null?

Reply:
@medfreeman I believe, but will double check, that it could be null while mounting (prior to render) and right after unmount, so it's safer to make the null check, but haven't looked in a while, so I could be wrong

Reply:
it makes sense. thx @mattcosta7